### ISSUE TITLE
ForkingPickler can't pickle

### ISSUE THREAD

**Issue by biswajitSM** [micro-manager/pycro-manager#9]
Hi,
I was trying out the pycro-manager with democamera config in microanager-gamma (14May nightly build) in a jupyter notebook with the following codes as documented.
```python
from pycromanager import Acquisition, multi_d_acquisition_events

with Acquisition(directory=r'C:\Users\biswajitpradha\Reseach\analysis\PycroManager\data', name='acquisition_name') as acq:
    events = multi_d_acquisition_events(
                                    num_time_points=4, time_interval_s=0,
                                    channel_group='channel', channels=['DAPI', 'FITC'],
                                    z_start=0, z_end=6, z_step=0.4,
                                    order='tcz')
```
But it showed an error: 
```python
AttributeError: Can't pickle local object 'Acquisition.__init__.<locals>.event_sending_fn'
```
full error pic:
![image](https://user-images.githubusercontent.com/12598051/82054963-1517f400-96c0-11ea-8b48-e9831bcd49c4.png)

Do you know what's going wrong here?

Thanks, Biswajit

**Comment by henrypinkard** [comment]
Appears to be something specific to python launching new processes on Windows. I just pushed a new version that may fix this. Upgrade to version 0.0.3 with `pip install pycromanager --upgrade` and give it a try

**Comment by dpshepherd** [comment]
Hi @henrypinkard ,

Thanks for the hard work on this! Same issue here. I just tried to install version 0.0.3 and it isn't found. pip install reports that only 0.0.1 and 0.0.2 are available.

Looking forward to trying the package out on a couple of our rigs,
Doug

**Comment by henrypinkard** [comment]
Whoops, look like I somehow forgot to push. Its up now. Let me know how it goes

**Comment by dpshepherd** [comment]
This code,
```
from pycromanager import Acquisition, multi_d_acquisition_events

with Acquisition(directory=r'C://testing//data', name='acquisition_name') as acq:
    events = multi_d_acquisition_events(
                                    num_time_points=4, time_interval_s=0,
                                    channel_group='channel', channels=['DAPI', 'FITC'],
                                    z_start=0, z_end=6, z_step=0.4,
                                    order='tcz')
```

used with version 0.0.3 now gives the following error:

`TypeError: no default __reduce__ due to non-trivial __cinit__`

full error picture:
![image](https://user-images.githubusercontent.com/26783318/82094884-c44dce80-96b2-11ea-888f-1ca3c18d3835.png)

**Comment by henrypinkard** [comment]
looks like this is a quirk with how windows handles multiple processes. For the time being, you can fix this by putting your whole script in a block with:

```
if __name__ == '__main__':
     with Acquisition...
```

I'm going to keep playing around to see if I can find a more durable solution

**Comment by henrypinkard** [comment]
I guess the above is the best solution possible due to how `multiprocessing` is implemented on windows. This should be fixed by #18, and I just pushed version 0.2.0 to pip that has these changes. 

You'll need to get the next MM nightly build after this PR merges (https://github.com/micro-manager/micro-manager/pull/805).

Closing for now since I think this is fixed. Open a new one if you run into something else
